BootloaderCorePkg/Stage2/Stage2.c

```diff
@@ -339,6 +347,14 @@ S3ResumePath (
   // Call the board notification
   BoardInit (EndOfStages);
 
+  // Stop timer so boot retries stop
+  if (PcdGetBool (PcdSblResiliencyEnabled)) {
+    StopTcoTimer ();
+    if (GetBootMode () != BOOT_ON_FLASH_UPDATE) {
+      ClearFailedBootCount ();
+    }
+  }
```

Comment:
Does the spontaneous corruption is supported in S3 path?

Reply:
Yes.